### PR TITLE
Feature/product tabs faqs

### LINKED ISSUE #973: [S71.1]:Product Specific Tab
### Link to Epic

Epic: https://github.com/Open-Study-College/osc/issues/911

### Description

n/a

### Linked Tasks

- [ ] #974 
- [ ] #975 
- [x] #976 
- [ ]
- [ ]

### LINKED ISSUE #974: Need to take some default/global data such as Awarding body/FAQs
### Link task to story

Story: https://github.com/Open-Study-College/osc/issues/973

### Description

n/a

### LINKED ISSUE #975: FAQs
### Link task to story

Story: https://github.com/Open-Study-College/osc/issues/973

### Description

should be either free edit or dropdown of selectable items

n/a

### PR DESCRIPTION
**Pull request by stevenbister**
- Closes #973
- Closes #974
- Closes #975 

## 📝 Description

Adds an item to the tabs component in Sanity to select a reference from a list of the faqs documents as well as an option to add custom faqs on a case by case basis. Displays the content in the tab on the ecommerce site.

https://github.com/Open-Study-College/osc/assets/23461173/568a323c-3aab-418f-b470-bac146a6cd9d

## ⛳️ Current behavior (updates)

- All tab content is manually added

## 🚀 New behavior

- Adds `tabFaqsItem` to tabs array in Sanity schema to reference `faqs` document
- Adds `FaqsContent` subcomponent to `Tabs`

### CODE REVIEW

**Review by mattwendzina — COMMENTED** [review]
One small comment, but other than that looks good mate!

File: packages/osc-ecommerce/app/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -123,3 +127,26 @@ const AwardingBodyContent = (props: TabContentProps) => {
...
+const FaqsContent = (props: TabContentProps) => {
+    const { tabItem } = props;
+
+    return (
+        <article className="o-container">
+            <h2 className="u-mb-0">{tabItem?.title}</h2>
+
+            <Accordion type="single" collapsible>
+                {tabItem?.faqs?.map((faq, i) => (
+                    <AccordionItem key={i} value={`${i}`}>
```
[the remark this answers is not in the document]
Comment (stevenbister) [reply packages/osc-ecommerce/app/components/Tabs/Tabs.tsx:140]: Ah yeah, it needs to be a string but the `i` is a number so I'm just coercing it. Admittedly looks pretty weird 😅

[the remark this answers is not in the document]
Comment (mattwendzina) [reply packages/osc-ecommerce/app/components/Tabs/Tabs.tsx:140]: Aha! Makes sense lol 😄

**Review by mattwendzina — APPROVED** [review]
LGTM

**Pull request merged**